### ISSUE TITLE
Raspberry Pi Issue.

### ISSUE THREAD

**Issue by gamer167** [coder/coder#14545]
Hi!

I was going to use this to access my files and I tried to install it. It "Installs", i do the command, and it errors. The error is about there being no unit to use. I used the Curl command to install as instructed by the Tailscail docs and this is what happens.

Please help!

**Comment by gamer167** [comment]
Also, I try reconfiguring it and the file is not there.

**Comment by stirby** [comment]
Hey, can you elaborate a bit? You're trying to get Coder running on a Raspberri Pi? Which Tailscale docs are you following? 

If I understand correctly, you're encountering errors on our `install.sh` script?

**Comment by gamer167** [comment]
Correct. The like to the tailscail doc im using is: https://tailscale.com/kb/1166/vscode-ipad
